.github/workflows/semgrep.yml

```diff
@@ -28,11 +28,11 @@ jobs:
 
       - name: Pull request scan
         if: ${{ github.event_name == 'pull_request' }}
-        run: semgrep scan --config=auto --verbose --time --error --baseline-commit ${{ github.event.pull_request.base.sha }}
+        run: semgrep scan --config=auto --verbose --time --error --baseline-commit ${{ github.event.pull_request.base.sha }} --exclude-rule yaml.docker-compose.security.writable-filesystem-service.writable-filesystem-service
```

Comment:
Did we check if security had concerns with this or if they had seen this before? I think we can still merge but would be good to ask them IMO.

Reply:
As all other project we found don't even have any semgrep validatoin for their docker-compose.yaml file I think we can get away with this one. All other rules are enforced.